libraries/Microsoft.Bot.Builder.Ai.Translation/Translator.cs

```diff
@@ -136,71 +110,52 @@ public Translator(string apiKey, HttpClient httpClient = null)
             }
 
             // body of http request

```

Comment:
I don't think this comment is correct now, it was related to the old variable 'body'